test/framework/integration_framework/integration_test_framework.hpp

```diff
@@ -113,79 +120,45 @@ namespace integration_framework {
 
   template <typename Lambda>
   IntegrationTestFramework &IntegrationTestFramework::sendTx(
-      iroha::model::Transaction tx, Lambda validation) {
+      shared_model::proto::Transaction tx, Lambda validation) {
     log_->info("send transaction");
-    // deserialize
-    auto pb_tx = iroha::model::converters::PbTransactionFactory().serialize(tx);
-    // send
     {
       google::protobuf::Empty response;
       iroha_instance_->getIrohaInstance()->getCommandService()->ToriiAsync(
-          pb_tx, response);
+          tx.getTransport(), response);
     }
     // fetch status of transaction
-    iroha::model::TransactionResponse::Status status;
+    std::shared_ptr<shared_model::proto::TransactionResponse> status;
```

Comment:
It does not seem necessary to create a shared pointer and code block below. Maybe it is possible to write it without code block and create transaction response model directly from `protocol::ToriiResponse` ?